ETC.tsv

```diff
@@ -9459,9 +9459,9 @@ ETC_20150317_009458	Mural
 ETC_20150317_009459	Runaway Adelaide (1)
 ETC_20150317_009460	Seal Device
 ETC_20150317_009461	Bags of the explorer
-ETC_20150317_009462	Soulf of Explorer Varkis
-ETC_20150317_009463	Solder's Dead Body
-ETC_20150317_009464	
+ETC_20150317_009462	Soul of Explorer Varkis
+ETC_20150317_009463	Soldier's Dead Body
+ETC_20150317_009464	Altar of Sviesa
```

Comment:
Altar of Sviesa > Sviecia Altar

I've done some of the names you've done too in my own PR https://github.com/Treeofsavior/EnglishTranslation/pull/503